spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/DeploymentState.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.module;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Deployment states for modules and streams. Unless indicated, these states
+ * may represent the state of<ul>
+ * <li>an entire stream</li>
+ * <li>the global state of a deployed module as part of a stream</li>
+ * <li>the state of a particular instance of a module, in cases where {@literal module.count > 1}</li>

```

Comment:
For this JavaDoc, was the intent for module.count to be in fixed width font or the same font as the rest of the text?